src/mame/video/k057714.cpp

```diff
@@ -121,6 +145,20 @@ void k057714_device::device_stop()
 #endif
 }
 
+void k057714_device::set_pixclock(const XTAL &xtal)
```

Comment:
There’s no point taking an `XTAL` if you don’t validate it.  You need to do something like `xtal.validate(std::string("Setting pixel clock for ") +tag());` (see example in src/emu/screen.h).